web/themes/custom/digital_gov/digital_gov.theme

```diff
@@ -408,3 +408,33 @@ function digital_gov_preprocess_node__landing_page__job_board(&$variables, $hook
   }
   $variables['job_board'] = $filtered_array;
 }
+
+/**
+ * Implements hook_preprocess_pager().
+ */
+function digital_gov_preprocess_pager(&$variables) {
+  // Appends fragment to pager links on /events to maintain scroll position.
+  $current_path = \Drupal::service('path.current')->getPath();
+  $alias = \Drupal::service('path_alias.manager')->getAliasByPath($current_path);
+
+  $anchor = '#events-past';
+
+  // Ensure we are only affecting the pager on the /events page.
+  if ($alias === '/events' && !empty($variables['items'])) {
+    // Iterate through all items in the pager.
+    foreach ($variables['items'] as $key => &$item) {
+      // Handle numbered pages separately from navigation items.
+      if ($key === 'pages') {
```

Comment:
We should also append it to the `next`, `previous`, `first`, and `last` links, but you have to check that the key exists first.